core-tck/pom.xml

```diff
@@ -111,25 +96,18 @@
             <scope>test</scope>
         </dependency>
         
-        <!-- Payara Arquillian Dependencies -->
-        <dependency>
-            <groupId>fish.payara.arquillian</groupId>
-            <artifactId>arquillian-payara-server-managed</artifactId>
-            <version>${payara.arquillian.version}</version>
-            <scope>test</scope>
-        </dependency>
         <dependency>
             <groupId>org.jboss.arquillian.junit5</groupId>
             <artifactId>arquillian-junit5-container</artifactId>
             <scope>test</scope>
         </dependency>
-        <!-- Commented out for now since Jersey Client seems to fail 
+        <!-- Commented out for now since Jersey Client seems to fail
         <dependency>
             <groupId>fish.payara.arquillian</groupId>
-            <artifactId>payara-client-ee9</artifactId>
+            <artifactId>payara-client-ee11</artifactId>
```

Comment:
Should we need to maintain the commented dependencies?

Reply:
This one I say yes, as ideally we Should™ be using our own client dependencies.
The other ones I've left in because there are follow-up tasks to review whether or not they're needed for Core Profile certification - TCKs like CDI have been split up and I don't know if our existing CDI TCK runner is actually running CDI Lite or if we need anything extra from this Core TCK